### PR TITLE
Issue #23: Implement the ICD

### LINKED ISSUE #23: Implement the ICD for the Publisher
Update the `publisher` function to send messages that are in the format of the Talos ICD.

### PR DESCRIPTION
**Pull request by DevanTurtle7**
Implemented the ICD and the 2 documentation functions (handshake and polar pan). Used C Types to ensure that the number of bytes will be correct.

closes #23

### CODE REVIEW

File: icd_config.py
Comment (nlc4447) [inline icd_config.py:1]: looks like __ int __"should probably be __ init __ if it's a constructor

**Review by nlc4447 — COMMENTED** [review]
Besides the constructor typo in icd_config.py this looks good.

File: icd_config.py
Comment (DevanTurtle7) [reply icd_config.py:1]: This is not a constructor since this is an enum. This is for casting the enum to an integer.

**Review by nlc4447 — APPROVED** [review]

**Comment by bl2060** [comment]
The only comment I have about the code is that the Command ID was intended to be unique across all messages, not just messages of a specific command type. That being said, this gives us significantly more possible IDs to cycle through. *Probably* won't be a big deal either way, only thing I could think of it it would take longer to verify the Command ID isn't a repeat (although we would need to actually test to see if that's significant).

I'd want to double check that the messages come through on the client as expected before merging, but other than that I think it's good to go.

**Comment by nolanporter77** [comment]
PR Confidence: 3

**Pull request merged**